akka-projection-r2dbc/src/it/scala/akka/projection/r2dbc/TestConfig.scala

```diff
@@ -10,37 +10,28 @@ import com.typesafe.config.ConfigFactory
 object TestConfig {
   lazy val config: Config = {
     val defaultConfig = ConfigFactory.load()
-    val dialect = defaultConfig.getString("akka.projection.r2dbc.dialect")
+    val dialect = defaultConfig.getString("akka.persistence.r2dbc.connection-factory.dialect")
 
     val dialectConfig = dialect match {
-      case "postgres" =>
-        ConfigFactory.parseString("""
+      case "h2" =>
+        // we need a new file each time or else we get file lock problems
```

Comment:
what is the reason for using file and not mem in test? 
is it an easy way to have a clean db for each test?
or are tests running in parallel and mem would use same for several tests?

Reply:
Mostly because thought that was closer to a realistic use of H2 with projections, at least while working on this. inmem is so much faster since it does no IO so it could hide issues.